.gitignore

```diff
@@ -9,3 +9,4 @@ pentagon.egg-info
 venv
 dist
 build
+.pypirc
```

Comment:
this is to avoid a `dirty` and ignore what the ci pipeline creates for releases.